### PR TITLE
Feat/cli cdp management

### LINKED ISSUE #7: Introduce scripts to support CDP Management
Add scripts to fetch the state of all CDPS owned by the org's multisigs. This should include all necessary information to aid decision making:
- CDP ID
- Collateral amount
- Debt amount
- ICR
- TCR
- % of Total Collateral
- % of Total Debt
- Place in sorted CDP list

### PR DESCRIPTION
**Pull request by petrovska-petro**
close #7 

run, sample:

```
brownie run scripts/cdp_management_lens.py --network sepolia-fork main 0x491657f0b1e70fB175B3d079b26EcC962AfFEA6C
```

### CODE REVIEW

**Comment by sajanrajdev** [comment]
@petrovska-petro seems like this PR includes the changes from the UniV3 helpers. Do you want to rebase to that branch or perhaps claw back the UniV3 changes?

**Comment by sajanrajdev** [comment]
## Review

Introduces a script that using an address as input, outputs all of the active CDPs owned by the address with the following details for each: CDP ID, Collateral, Debt, ICR, cdp collateral vs total cdp's owned (%) and cdp debt vs total cdp's owned (%). In addition, it outputs the current ICR and price as observed by the system in real time.

LGTM

- Clean and concise output: 
image
- Confirmed its accuracy: 
image

Will just wait on fixes to wrong base before approving.

**Comment by petrovska-petro** [comment]
@sajanrajdev base has being changed

**Review by sajanrajdev — APPROVED** [review]

File: scripts/cdp_management_lens.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    for cdp_id in cdps_safe_owned:
+        C.print(f"[green]Inspecting cdp id: {cdp_id}\n[/green]")
+        (
+            cdp_id_debt,
+            cdp_id_coll,
+            _,
+            _,
+            _,
+            _,
+        ) = safe.ebtc.cdp_manager.Cdps(cdp_id)
```
Comment (rayeaster) [inline scripts/cdp_management_lens.py:55]: here we might use `getSyncedCdpCollShares()` & `getSyncedCdpDebt()`

**Pull request merged**